gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -226,7 +261,74 @@ The following example shows how to permit the A10 GPU device and A10-24Q vGPU de
 
 #. Modify the ``KubeVirt`` custom resource like the following partial example. 
 
-   Add 
+   .. code-block:: yaml
+
+      ...
+      spec:
+        configuration:
+          developerConfiguration:
+            featureGates:
+            - GPU
+            - DisableMDEVConfiguration
+          permittedHostDevices: # Defines VM devices to import.
+            mediatedDevices: # Include for vGPU 
+            - externalResourceProvider: true
+              mdevNameSelector: NVIDIA A10-14Q
+              resourceName: nvidia.com/NVIDIA_A10-14Q
+      ...
+
+   Replace the values in the YAML as follows:
+
+   * ``mdevNameSelector`` and ``resourceName`` under ``mediatedDevices`` to correspond to your vGPU type.
+
+   * Set ``externalResourceProvider=true`` to indicate that this resource is provided by an external device plugin, in this case the ``sandbox-device-plugin`` that is deployed by the GPU Operator.
+
+Refer to the `KubeVirt user guide <https://kubevirt.io/user-guide/virtual_machines/host-devices/#listing-permitted-devices>`_ for more information on the configuration options.
+
+.. _add-gpu-passthrough-resources-to-kubevirt-cr:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Add GPU passthrough resources to KubeVirt CR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Update the KubeVirt custom resource so that all GPU and vGPU devices in your cluster are permitted and can be assigned to virtual machines.
```

Comment:
```suggestion
Update the KubeVirt custom resource so that all GPU passthrough devices in your cluster are permitted and can be assigned to virtual machines.
```